packages/enzyme/src/ShallowWrapper.js

```diff
@@ -122,25 +122,26 @@ class ShallowWrapper {
           instance.componentDidMount();
         });
       }
-      privateSet(this, NODE, getRootNode(this[RENDERER].getNode()));
-      privateSet(this, NODES, [this[NODE]]);
-      this.length = 1;
+      this.setNodesInternal(getRootNode(this[RENDERER].getNode()));
```

Comment:
this makes `setNodesInternal` a public function; it's important to minimize the publicly observable API. Can this be moved to a module-scoped function?